### DISCUSSION TITLE
Attachments are not updating on website

### DISCUSSION THREAD

**Discussion by AKA1973** [community/community#41824] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

Hello forum.  This is the first time posting a question and hoping this is the correct place to ask.

For the past year, I've been able to upload my attachments to the website for users to view or download.  When I tried to update the documents for November 2022, the attachments are viewable when I run Visual Studio, but when I load the webpage, October 2022 information still appears.  When I check my Github account to see if the images loaded, they are the current documents and show that they have replaced the previous document.  Below are the weblinks I hope work so you can see this.

**Link that shows the images were uploaded and work in Github:**
https://github.com/AKA1973/AmerKorAsian/tree/main/images

**Link that shows how the actual webpage should look and is visible when users visit the website:**
https://aka1973.github.io/AmerKorAsian/HomePage.html

If you need to see the code, I believe it's viewable, but can reply with it here if you need.  Thanks in advance for your help and I hope I've posted this in the correct section.

Ryan

**Comment by DSDmark** [comment]
Hi @AKA1973,

if I was understanding correctly. Then, you are having an issue with caching on your website. When you upload new documents to your website and run the site locally in Visual Studio, the new documents are displayed because they are being loaded from your local machine. However, when you visit the website from a browser, the old documents are displayed, Right.

Then, you can try clearing the cache for your website in your browser.

And also you could try to **run a workflow**. It will update your website with current source code which you have on your repo.

![image](https://user-images.githubusercontent.com/85744816/207869120-a20723bc-707a-4087-8239-c750bd24d9d3.png)

**Comment by AKA1973** [comment]
Hello DSDmark.  Thank you very much for replying back with possible solutions to fix the problem.

Just some background on me.  I'm not a coder, but spent time to figure out how to create a webpage and include features like adding documents and allowing people to email me directly from the website.

I just have a couple of follow-up items because the one fix you mentioned isn't an option for me.  Below is the screenshot of the same page you sent, but I don't have the **"Run Workflow"** option like you have.

![image](https://user-images.githubusercontent.com/88971334/208016880-4e94bc8b-c689-448b-91a3-c09611a4509d.png)

1) I also tried re-saving a couple of the documents, but they still don't show up on the website.
2) Beginning with the documents I uploaded for November 2022, the workflow runs no longer say by **github-pages bot**, but by **AKA1973**.
3) When I click the **Deployments** option on the above page, I go to the deployed page, which I've screenshot below.

![image](https://user-images.githubusercontent.com/88971334/208017505-cfd2988e-4d8a-48cd-b5cb-f3a9fcf85afa.png)

When I click the view **"deployment option"** on the above page, the website that appears is correct, but isn't the deployed website that people see.

https://aka1973.github.io/AmerKorAsian/

How do I get the View Deployment website to be the website for my domain?

**Comment by radar-blip** [comment]
I think what is happening is, you want HomePage.html to show first, right? 
[HomePage](https://aka1973.github.io/AmerKorAsian/HomePage.html)

But by default, Github Pages will use the index.html (which is happening).
[index.html](https://aka1973.github.io/AmerKorAsian/index.html)

If that is the case, you will need to switch those around somehow.

**Reply by AKA1973** [reply]
Thank you for replying, but the issue has to do with the documents not updating on the website.

**Reply by DSDmark** [reply]
@AKA1973, can you add screenshot which document is not updating.

**Reply by AKA1973** [reply]
Hello again DSDmark.  None of the chart or monthly numbers that are added to the website every month are updated.  I even re-uploaded the 2 main data sets a few hours ago to see if they would update.

![image](https://user-images.githubusercontent.com/88971334/208038810-345e0223-b2a2-4b61-bf5e-45f3fc714b6f.png)

If you go directly to my website:  www.amerkorasian.com

And either click on the charts or on an actual pie on the right side of the page, the documents that appear should also include the November 2022 month-end numbers as you can see are sitting in the repository in Github.
 
![image](https://user-images.githubusercontent.com/88971334/208039223-5840da15-82d1-4501-9834-520924013e57.png)

If you click on the link below the charts on the website, it will take you to another page that has additional information that also isn't correct because it still shows the October information.

**Comment by DSDmark** [comment]
@AKA1973, Ok, So, as I understand. Your  ```PRIOR 6-MONTHS PERFORMANCE BY PIE``` image is not updated. But it's already updated in your repo, which is this one.

![image](https://user-images.githubusercontent.com/85744816/208047712-28902843-334f-4cac-9e8e-cce12baa71de.png)

---

So, I guess first try these steps.

click on **action tab** and click on your workflow file like this.

![image](https://user-images.githubusercontent.com/85744816/208046123-0366be78-674f-4787-acb1-241727aa4c9e.png)

Then rerun all jobs and wait, then check again your site.

![image](https://user-images.githubusercontent.com/85744816/208046236-2138e21d-2df9-41aa-9411-fa312a6ba37d.png)

**Reply by AKA1973** [reply]
All the information I sent in previous message hasn't been updated.

![image](https://user-images.githubusercontent.com/88971334/208051017-af6c0998-fa90-418f-b82b-8ead90393464.png)

![image](https://user-images.githubusercontent.com/88971334/208050626-a65d1810-7256-4de1-abd9-9fe8a6a4b6f2.png)

I did as you suggested, but the website still shows October 2022 numbers.
![image](https://user-images.githubusercontent.com/88971334/208051333-21e1e42e-1d73-46aa-85b2-951019a22fa4.png)

![image](https://user-images.githubusercontent.com/88971334/208051443-c5e00a9e-23c9-4674-9409-a29aa2b77760.png)

![image](https://user-images.githubusercontent.com/88971334/208054830-06eac13c-513e-4129-9dbd-5afb2517cb27.png)

![image](https://user-images.githubusercontent.com/88971334/208055549-dc8a7197-1d9e-4bca-9cd8-b2ad48a7b121.png)

![image](https://user-images.githubusercontent.com/88971334/208056491-46422dd5-22ed-4d2a-a16c-9153a75ad4a7.png)

And the site still shows October numbers :(

**Reply by DSDmark** [reply]
Hey @AKA1973, are you sure images not updated.

Check here again
https://aka1973.github.io/AmerKorAsian/HomePage.html

**Reply by DSDmark** [reply]
I think. I get it. The main reason why doesn't your images update. Because of, there are two different websites. 

First with that URL:- [www.amerkorasian.com](http://www.amerkorasian.com/)

![image](https://user-images.githubusercontent.com/85744816/208095840-c59f7b26-1858-4080-bd74-854530faac16.png)

And second one with that URL :- https://aka1973.github.io/AmerKorAsian/HomePage.html

![image](https://user-images.githubusercontent.com/85744816/208096238-bd7c7aed-7991-4e64-9ea2-c21bfac9000d.png)

**Comment by DSDmark** [comment]
@AKA1973,

as you can see in down below images, both sites and the Same images. But different destination.

![image](https://user-images.githubusercontent.com/85744816/208099992-9131f73c-611c-4351-8fe4-44ad1c0a8f15.png)

First one is using **github-pages** and second one is using **Netlify**.

**Reply by DSDmark** [reply]
I'm not sure. About that. But, did you use Netlify by any chance to deploy your site in the past.

**Reply by AKA1973** [reply]
OMG!  Yes, I used Netlify to launch the website initially, but learned that Github is more robust and better to use especially for someone like me that isn't a coder.

My question to you is, why is this issue happening now?  Until I tried uploading the documents for November 2022, the updates correctly displayed on the website.  From what I could see, the only thing that changed was the page builds and deployment went from github-pages to AKA1973.

What do I need to do to correct this?  Maybe it would be better to discuss this thru email.  Can you email me at ryan.holland@amerkorasian.xom?

We can also continue here as well.  Whatever is more convenient for you.

Ryan

**Reply by DSDmark** [reply]
I guess, on that time. When you deploy your sites on ***Netlify*** and ***github-page***. Both were same sites with source code, same images and assets. So, you didn't notice it. 

You can simply remove your deployment from any one platform which you want (suppose you remove from Netlify). Then, you can focus on one deployment. I see you have a custom domain name. If you want to use that custom domain on your ***github-pages***. Then, you can add that domain by follow these steps. 

[**GitHub Docs**](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site). 

And if you need any help, you can ask here. We will try to help you.

**Reply by AKA1973** [reply]
Hello again.  I spent the last day+ trying to figure out how to remove deployments to Netlify, but was unsuccessful.  As you can see from the below screenprint, I suspended deployments to Netlify and thought that would fix the problem temporarily until figuring out how to change the code but didn't work.  The website still shows October 2022 information.  There is an option to uninstall Netlify, but I'm not sure if that option can be reversed, I only suspended for the time being.  Will uninstalling Netlify cause any other errors?

![image](https://user-images.githubusercontent.com/88971334/208280528-72128637-94a5-43ff-8e97-6032687a7afb.png)

I also followed the instructions from the Github Docs link you provided, but says that Setup failed because is already installed on laptop.
https://help.dyn.com/how-to-use-binds-dig-tool/

![image](https://user-images.githubusercontent.com/88971334/208281789-a1a65fe3-1792-475f-b045-40667bd04495.png)
![image](https://user-images.githubusercontent.com/88971334/208281804-bd618e09-ffb2-41e8-a87f-4d3dd62fd87c.png)
![image](https://user-images.githubusercontent.com/88971334/208281819-79843e52-1eb5-4337-9c1c-c06f6ae954af.png)

I even followed the steps to open the command prompt and tried to complete the BIND process, but also didn't work correctly.
![image](https://user-images.githubusercontent.com/88971334/208282159-11d3c977-5f37-4367-a9a8-4c68d006da40.png)

It seems like this should be an easy step to resolve, but because I'm not a coder, I'm not exactly sure what to look for to either remove Netlify or re-direct to my custom domain.

Lastly, the domain name associated with my account as you know is aka1973.github.io.  I tried adding the following CDN names, but all errored out:

![image](https://user-images.githubusercontent.com/88971334/208282494-9e0fafb8-9861-42c9-918f-23dabddc1d35.png)
www.amerkorasian.com
![image](https://user-images.githubusercontent.com/88971334/208282570-6590bf02-5fdc-40d2-87ce-8301ce5b257d.png)

amerkorasian.com
![image](https://user-images.githubusercontent.com/88971334/208282538-e88fd964-93e6-4bfe-b08d-4ce8c6c81e05.png)

All errors bring me back to the Github Docs link you provided, but can't find any information on how to either remove Netlify or redirect from Netlify.

What am I doing wrong that Github is not letting me add my custom domain on my Gidhub-pages?

**Comment by DSDmark** [comment]
Everything's is expected as it should be. You don't have idea to how to do that.

For removing the site on Netlify. First, you need to `login` on [**Netlify**](https://app.netlify.com/sites/heartfelt-blancmange-64d839/settings/general).

After that, click on `Deploy` option. Then, `Deploy settings`. As you can see in the Down below image.

![image](https://user-images.githubusercontent.com/85744816/208287372-1c088de7-10a4-4493-b379-be8d8ea64903.png).

And then, In `Gerenal` tab. You find the `Denger zone`. Click on that and `Delete the site`.

![image](https://user-images.githubusercontent.com/85744816/208287424-db69bf9d-bb36-4e8f-b4f0-b056fc65a313.png)

**Comment by DSDmark** [comment]
For Adding the Domain name in `Github-Page`. First, you need to create a file which name is `CNAME` by clicking on `Add file` button on your repository. As it's showing in down below image.

![image](https://user-images.githubusercontent.com/85744816/208288266-e504302f-011a-4773-a070-dc3f1f4fc8a7.png).

Inside the file. You can see the is only containing the `Domain name`, Ok.

![image](https://user-images.githubusercontent.com/85744816/208288359-177a60f4-aedf-4e1f-a782-ec1b12dcff15.png)

After that, if you're done that. Then, further step take long time for configuration. Because, its relate to your `Domain name` provider. So, As you guess open, you `Domain name` provide penal and then, there are two things you need to config.

> **Note**:- Further settings may change. its depenting up on you `Doman` provider.

I'm using `Name.com` for that.

Inside you `Domain name` provide penal. You need to find `DNS Record` or `Manage DNS` or may it something's else. Once you find out `DNS` section. You need to add two `DNS Record`.
1. `A` Record.
2. `CNAME` Record 
As you can see in the down below image. 

![image](https://user-images.githubusercontent.com/85744816/208289199-331c1b5d-c920-4769-9e1f-40bd16b757fb.png)

After that, I mean after 48 hours later, you can check again your `Domain name` like this:-

![image](https://user-images.githubusercontent.com/85744816/208289268-13de24e8-63cb-4859-a131-97f79e36ee49.png)

**Comment by AKA1973** [comment]
Thank you again for your help.  As suggested, I logged back into my Netlify account to delete the website, but couldn't because as you can see from the below images, I already deleted the site from Netlify.

![image](https://user-images.githubusercontent.com/88971334/208325845-63ceb177-aa4e-4192-96ca-9112c485ae53.png)

Since I saw that Github was still a connected account with Netlify, I disconnected the link which should prevent any communication between Github and Netlify.  At least from the Netlify site.
![image](https://user-images.githubusercontent.com/88971334/208325669-5dba38cf-9d88-47c0-8281-0cedbb7ec92d.png)

![image](https://user-images.githubusercontent.com/88971334/208325726-06830dcb-4218-43e9-8ac5-e89053de1cf9.png)

I think I had to remove the site from Netlify when I moved everything over to Github because my website wouldn't deploy while still being connected to Netlify.

![image](https://user-images.githubusercontent.com/88971334/208326527-9f78a307-1500-4e35-9685-24f1218a89a9.png)

I believe I did the above step correctly and the reason why mine doesn't show the domain name is because it's configuring right now.  I will look for how to do the next step in Github while I wait for the domain name to populate in the CNAME file.

**Comment by AKA1973** [comment]
Step 1 is done and now trying to complete Step 2 - adding the 2 DNS records.
![image](https://user-images.githubusercontent.com/88971334/208327186-fa757ec6-aad6-4c65-b884-21925fe8ac5a.png)

**Reply by AKA1973** [reply]
Tried adding the 2 DNS records to my Google Domains account, but not sure where to get the IP address needed for the data field.

![image](https://user-images.githubusercontent.com/88971334/208371067-8ea839d6-9393-4c52-85da-f5bcba249eb2.png)

I followed the Github Docs instructions on making sure the DNS record is configured correctly, but when I tried to complete Step 7, I kept getting an error message which I guess means that the DNS record isn't set-up correctly.

![image](https://user-images.githubusercontent.com/88971334/208372255-f3c10be1-826f-40a1-b04d-88a448e16c38.png)

I was able to update the DNS records immediately on the Google Domains site.
![image](https://user-images.githubusercontent.com/88971334/208374405-57c85ecf-a14a-45d1-99a6-3ca64cca27c9.png)

But the website still shows October 2022 information.  Do I need to wait up to 48 hours for the changes to take effect?

**Reply by DSDmark** [reply]
I apologize, @AKA1973, for uploading the wrong images. Its looks this.

![image](https://user-images.githubusercontent.com/85744816/208425098-f0e594e1-92fe-4500-a01f-58329eff98f6.png)

As its described in these docs:- [GitHub docs](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain)

**Reply by DSDmark** [reply]
And also after that, don't forget to config into your settings.

![image](https://user-images.githubusercontent.com/85744816/208425612-388a5324-9897-4060-81a6-74b4d10e9aaf.png)

If it shows error. Then, wait for 48 hours. Sometimes it takes time to config.

**Reply by DSDmark** [reply]
And also for removing `Netligy`. Open your settings and click on `Application` section. Here you can remove it.

![image](https://user-images.githubusercontent.com/85744816/208426796-62b38943-24be-4497-ad48-0b5b52464386.png)

**Comment by AKA1973** [comment]
No worries DSDMark.  I appreciate the help in trying to fix this.  What I need done would probably take someone who knows what they're doing about 5 minutes and it seems like I'm just about at the end and will have the correct information on the website shortly.

**I updated the Host and Type A information using the same logic as in the example which you will see in the screenshot below.**
![image](https://user-images.githubusercontent.com/88971334/208551421-8e787a1b-b0ae-4bee-aa49-76e2ba13eb47.png)

**I changed the option from suspend to Uninstall Netlify**
![image](https://user-images.githubusercontent.com/88971334/208551706-36444442-ef39-4c3f-9da4-04c5c1e2da3b.png)

![image](https://user-images.githubusercontent.com/88971334/208551836-1f02c711-aba9-492c-b0a5-d3f95ae77a93.png)

And it appears that Google Domains is a pretty smart system.  The custom domain name includes the www. part so I'm not sure if I should change.  I thought I would wait 24-48 hours before trying to change to amerkorasian.com.  Your thoughts???
![image](https://user-images.githubusercontent.com/88971334/208552420-f07f6f83-04d3-4deb-8534-01973186777a.png)

**Reply by AKA1973** [reply]
Maybe the reason why the below isn't showing is for the same reason as we're fixing now, but adding here because both users and visitors are emailing me about when November numbers will be available on website.

I thought that adding the below snippet of code would add easily to the data file and be visible on the website.  But if you go to the website, you will not see this scrolling marquee.  Is this not visible because any new information added is being directed incorrectly and until we fix, nothing can be adjusted with the current code?

![image](https://user-images.githubusercontent.com/88971334/208563399-46e3a0dd-7246-4b9b-a9bc-28fb8ccc6a72.png)

**Reply by AKA1973** [reply]
I removed the above code but I can't undo removing the Netlify app.

![image](https://user-images.githubusercontent.com/88971334/208583706-d702c49b-ea78-4bd6-8ede-5975f44c8339.png)

As you can see from the above the website no longer works.  Below is also a screenprint from my Github account that says amerkorasian.com is not eligible for HTTPS at this time.

![image](https://user-images.githubusercontent.com/88971334/208584280-3cfb689a-37f1-45bc-b2f5-f5d20ab5e770.png)

What happened?  It seems like a simple issue is now becoming more complex.

**Comment by AKA1973** [comment]
![image](https://user-images.githubusercontent.com/88971334/208584514-7cd53ebc-bd16-4422-bfb7-90979c4596ab.png)

**Reply by DSDmark** [reply]
@AKA1973, try to check this.

![image](https://user-images.githubusercontent.com/85744816/208585640-08e640f5-ee79-409f-8280-fd625eef08e3.png)

**Reply by DSDmark** [reply]
Here we go https://www.dsdmark.systems/ if you click on that you can see on my domain everything's is fine.

It means you're messing something's again.

**Reply by DSDmark** [reply]
And please check the data which you want to update. On that URL   https://www.dsdmark.systems/ is updated or not

**Reply by DSDmark** [reply]
And if you're not getting your way to do it, then invite me on your repo.
Like this. 

![image](https://user-images.githubusercontent.com/85744816/208586757-9d9f1309-d538-4f0a-b180-4e28f0bd6060.png)

**Reply by AKA1973** [reply]
Hello.  I just added you as a collaborator.

**Reply by AKA1973** [reply]
Does Github have a chat window in it or maybe can do through gmail???

**Reply by AKA1973** [reply]
I just invited you to Google chat

**Reply by DSDmark** [reply]
@AKA1973, you site is working fine now https://amerkorasian.com/, so you can remove me as collaborator.

**Reply by DSDmark** [reply]
Check your data which you want to update. Is there or not, updated or not. Is everything is not fine, then, let me know. Ok.

**Reply by AKA1973** [reply]
![image](https://user-images.githubusercontent.com/88971334/208593472-696378ae-6052-45af-98de-4a912009e502.png)

Also won't work on my phone.

**Reply by AKA1973** [reply]
This is also happening
![image](https://user-images.githubusercontent.com/88971334/208593869-863c6f0a-b902-474e-84d4-8e28f078c59f.png)

**Reply by AKA1973** [reply]
re-invited you as collaborator

**Reply by AKA1973** [reply]
And these are the custom records I added in Google Domains to connect with Github.

![image](https://user-images.githubusercontent.com/88971334/208595069-6b0dfdd7-867c-438b-aada-0845455a1cff.png)

**Reply by AKA1973** [reply]
Checking to see if you're still looking at site.

**Reply by DSDmark** [reply]
Everything looks fine on my side. I think you need to wait, maybe `CNAME` it takes some time to config. 

![image](https://user-images.githubusercontent.com/85744816/208597452-38a246b3-f59a-47dc-8140-cda33ff31554.png)

You can do one thing. Check again with `www`. I mean add that with your URL.

**Reply by DSDmark** [reply]
@AKA1973, for now, your site is working fine on my side.

![image](https://user-images.githubusercontent.com/85744816/208598191-2759b456-ae4d-4327-bd8c-5cc4986cbfce.png)

but I suggest you to repeat these step:- https://github.com/orgs/community/discussions/38632#discussioncomment-4092967

**Reply by DSDmark** [reply]
When you do this step, select ***static HTML*** instead of **jekyll**

![200735302-a2dff9ad-a4b1-4af9-9128-7ba51b12329f](https://user-images.githubusercontent.com/85744816/208598692-2e15e3be-6a59-48c7-963e-5970eb58f019.png)

**Reply by AKA1973** [reply]
When I went to change to Static HTML, the below box was running.
![image](https://user-images.githubusercontent.com/88971334/208600366-3cfa686b-394f-4508-b3f4-668de4334d52.png)

**Reply by AKA1973** [reply]
Since it looks like it might take up to 24 hours for the certificate to be issued, I removed you as a collaborator.  Will of course let you know if something still isn't working correctly or the documents are still wrong.

Thank you very much for your help so far.

**Reply by DSDmark** [reply]
Do that. `Enforce HTTPS`.

And it just checking the `TLS`. Don't worry about that.

![image](https://user-images.githubusercontent.com/85744816/208601746-cc70ef41-1535-4542-b566-509a1105fab4.png)

**Reply by DSDmark** [reply]
> Since it looks like it might take up to 24 hours for the certificate to be issued, I removed you as a collaborator. Will of course let you know if something still isn't working correctly or the documents are still wrong.
>
> Thank you very much for your help so far.

Yes, And if you need. Then, don't hesitate to ask.

**Reply by AKA1973** [reply]
Quick question.  If the below page refreshes, does the whole process start again?

![image](https://user-images.githubusercontent.com/88971334/209042546-5fd2a92e-fbdc-4c71-b3ec-0c34ab9fcc00.png)


I have closed out of the page, but if refreshes, the process would not complete until 24-48 hours from now.

**Reply by DSDmark** [reply]
@AKA1973, the answer is no. it's a once time process, except you will not change the `RECORDS`. In actually record take time to config. Because it's set up or point your domain name with other domain name. 
for example:- if you have `xyz.com` it's your domain name and if you add `www` then it's become a subdomain like this `www.xyz.com`. So,

- `xyz` - is domain name
- `www.xyz` is subdomain name 

So when you config your records like `AAA` or `CNAME`. Then, it means you are to make change in domain settings. You can point `xyz` domain to your subdomain `www.xyz`. Even you can point your domain `xyz` to the official domain `www.github.com` or `www.google.com` or `www.stackoverflow.com`. But server is not able of that. So, but if you are able to config that. Then, you can point your domain to `www.github.com`.

Here is an example:- 
when you copy and paste my that domain in url `dsdmark.systems` its point to www.dsdmark.systems

**Reply by DSDmark** [reply]
😊 I hope you get your answer. If you get your response, you should close that question by clicking ✅ accept answer button, or you can ✍️ write the answer and accept your own answer.

> **Note**: if you close this question, then nobody will contribute to your question anymore. So, close this question when you satisfy your answer, or you can write your own question with a better explanation

**Reply by AKA1973** [reply]
Thanks again for your help.  I will wait until the system is done provisioning and if the website deploys without any errors, will close this ticket.

**Comment by AKA1973** [comment, marked as answer]
Looks like site is working, but the web address doesn't include www. before the site name.  I have included what I included in my DNS Custom records for your review.  Someone messaged me asking about the site because the web address didn't begin with www.

Is this a simple fix?

![image](https://user-images.githubusercontent.com/88971334/209293237-2e71554b-22a3-4c90-a595-30b5f37d9b85.png)

**Reply by AKA1973** [reply]
Another issue.

I created this site to be used on a mobile device, but I'm not able to open it on my iPhone because it says that Safari cannot establish a secure connection.  Is this just a timing issue and will work shortly?

**Reply by DSDmark** [reply]
@AKA1973, Sorry for replay so late. If you want `www` at the begging of your domain. Then, just simple add this to `CNAME` file like this:-

![image](https://user-images.githubusercontent.com/85744816/209429170-941a4632-6594-40c4-acdf-b0b5b7231925.png)

**Reply by DSDmark** [reply]
Here are some possible causes for this problem and steps you can take to troubleshoot:

  -  Check the website's SSL certificate: A secure connection error may occur if the website's SSL certificate is not properly configured or has expired. You can check the certificate by clicking on the lock icon in the address bar of your browser.

  -  Check the website's DNS configuration: Incorrect DNS configuration can also cause a secure connection error. You can try using a different DNS server or flush your DNS cache to see if that helps.

  -  Check your internet connection: A weak or unstable internet connection can also cause secure connection errors. Make sure you have a strong and stable connection before trying to access the website again.

 -   Check for any browser extensions or plugins that may be causing problems: Certain browser extensions or plugins can interfere with the secure connection. Try disabling these to see if it helps.

   - Check for any security software or firewall settings that may be blocking the website: Some security software or firewall settings may block certain websites from being accessed. Make sure that the website is not being blocked by your security software or firewall.

**Reply by AKA1973** [reply]
Hello again.  I hope you had a good holiday season and a nice New Year.

I was able to get the website to work on my iPhone by clearing the web history.  So the site now works on both a computer and on mobile devices.  Although I'm used to having the www. at the beginning of the web address, I'm ok for now with the website working without the www. in the front.  Maybe after completing the year-end information, I will circle back to see if I can add the www. at the beginning of the website address.

Thanks again for all your help and I will now close this ticket.
